Cargo.toml

```diff
@@ -90,6 +90,7 @@ wasi-crypto = ["wasmtime-wasi-crypto"]
 wasi-nn = ["wasmtime-wasi-nn"]
 uffd = ["wasmtime/uffd"]
 all-arch = ["wasmtime/all-arch"]
+posix-signals-on-macos = ["wasmtime/posix-signals-on-macos"]
```

Comment:
Not sure about the name. Bikeshedding is welcome. Possible alternatives: `macos-signals`, `macos-posix-signals`.

Reply:
I think it's probably fine to skip this feature on the `wasmtime-cli` crate since it's mostly the `wasmtime` embedding crate that wants this.

Reply:
It is useful for conditionally enabling `tests/all/custom_signal_handler.rs` on MacOS and running the tests with `cargo test --features=posix-signals-on-macos custom_signal_handler`
But indeed it's not needed for production. Should I remove it?